CONTRIBUTING.md

```diff
@@ -150,13 +150,13 @@ All pull requests are reviewed by another person. We have a bot,
 request.
 
 If you want to request that a specific person reviews your pull request,
-you can add an `r?` to the message. For example, [Steve][steveklabnik] usually reviews
+you can add an `r?` to the pull request message. For example, [Steve][steveklabnik] usually reviews
```

Comment:
```suggestion
you can add an `r?` to the pull request description. For example, [Steve][steveklabnik] usually reviews
```
Could also go for this but didn't want to make too big a change

Reply:
I prefer description since that's what GitHub uses